Automattic/wp-calypso
# A4A > Agency Tier: implement feature announcement modal

## Proposed Changes

This PR implements a feature announcement modal for the Agency Tier.

## Why are these changes being made?

* To show a feature announcement modal for 

## Testing Instructions

* Use the agency MC tool > Set the agency tier to undefined
* Open the A4A live link > Go to the Overview page > Verify that a modal is displayed as shown below & do not dismiss the modal. Also, verify it looks good on mobile view.
* Use the MC tool to update the agency tier to "Emerging Partner" > Verify that a modal is displayed as shown below. Dismiss the modal and refresh the page > Verify that it is not shown again.
* Also perform the test instructions here: https://github.com/Automattic/wp-calypso/pull/96486 and see if it works perfectly.

<img width="795" alt="Screenshot 2024-11-18 at 8 38 07 PM" src="https://github.com/user-attachments/assets/7d274ba8-e4bf-40e8-8034-450d226a0cba">

<img width="375" alt="Screenshot 2024-11-18 at 8 38 22 PM" src="https://github.com/user-attachments/assets/5e254221-8d6f-4a1c-8c0d-8b8b828a6db2">



## Pre-merge Checklist
